src/Core/Analysis/IAnalysisIssue.cs

```diff
@@ -39,6 +41,23 @@ public interface IAnalysisIssue
         string Message { get; }
 
         string FilePath { get; }
+
+        IReadOnlyList<IAnalysisIssueLocation> Locations { get; }
```

Comment:
This is the key data structure for secondary locations and the security vulnerabilities sprint to follow shortly.
We don't need flows for CFamily issues, but we'll have to rework practically every component we write this sprint when we move on to the security work. I think we could avoid some of that rework by modeling flows in the data model now, and treating the CFamily case as a degenerate case that only ever has zero or one flows.